app/views/contexts/section_summary_context.py

```diff
@@ -64,7 +64,10 @@ def __call__(
                 "summary_type": "SectionSummary",
                 "answers_are_editable": True,
                 "sections": {
-                    "groups": summary.get("groups"),
+                    self.current_location.section_id: {
+                        "title": title,
+                        "groups": summary.get("groups"),
+                    }
                 },
```

Comment:
Should be a list to match our schema, if you need `id`, add it into the object.

Reply:
Updated now 👍